src/app/register/register.component.ts

```diff
@@ -93,7 +94,7 @@ export class RegisterComponent {
         this.userProps.city = city;
         this.userProps.university = university;
         this.userProps.year_of_study = year_of_study;
-        this.userProps.date_of_birth = date_of_birth;
+        this.userProps.date_of_birth = date_of_birth + "-01-01";
```

Comment:
Because it's already a date field at the database. It's possible to change the field but it would require extra work at the server side which I would like to avoid.